docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc

```diff
@@ -1,26 +1,56 @@
-= SBOMs
+= Inspecting SBOMs
 
 :toc: left
 :icons: font
-:numbered:
 :source-highlighter: highlightjs
 
-== Software bill of materials
-A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM is a list of all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries might enable specific functionality, facilitate development, or serve some other purpose. 
+== Software Bill of Materials
```

Comment:
Why are `Bill` and `Materials` capitalized? They don't read as proper nouns, so I wouldn't think they ought to be capitalized. But perhaps I'm missing something?